### PR TITLE
feature: add russian language

### PR DESCRIPTION
**Pull request by RVShershnev**

### CODE REVIEW

File: src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+  "WebSocketConnection/ConnectionRequestHeaderAdd": "Добавить header",
+  "WebSocketConnection/ConnectionRequestHeaderRemove": "Удалить",
+  "WebSocketConnection/ConnectionRequestSubprotocolEnabled": "ВКЛЮЧЕН",
+  "WebSocketConnection/ConnectionRequestSubprotocolName": "ИМЯ",
+  "WebSocketConnection/ConnectionRequestSubprotocolAdd": "Добавить",
+  "WebSocketConnection/ConnectionRequestSubprotocolRemove": "Удалить",
+  "WebSocketConnection/EnableCompression": "Включить сжатие",
+  "WebSocketConnection/CompressionClientContextTakeover": "Захват клиентского контекста",
+  "WebSocketConnection/CompressionClientMaxWindowBits": "Максимальное количество битов окна клиента",
+  "WebSocketConnection/CompressionServerContextTakeover": "Server context takeover",
```
Comment (alexandrehtrb) [inline src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json:215]: Forgot translation

File: src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+  "WebSocketExchangedMessages/ClosingMessageContentDescriptionFormat": "closing, {0} байтов",
+  "WebSocketExchangedMessages/InstantDescriptionFormat": "HH:mm:ss, yyyy-MM-dd, 'GMT'z'h'",
+  "WebSocketExchangedMessages/SendMessage": "Отправить",
+  "WebSocketExchangedMessageDetail/SectionTitle": "Детали сообщения",
+  "WebSocketExchangedMessageDetail/MessageTypeWatermark": "(тип сообщения)",
+  "WebSocketExchangedMessageDetail/ContentWatermark": "(контент сообщения)",
+  "WebSocketExchangedMessageDetail/SaveToFile": "Сохранить как…",
+  "WebSocketClientMessage/NewMessage": "Новое клиентское сообщение",  
+  "WebSocketClientMessage/MessageTypeText": "Текст",
+  "WebSocketClientMessage/MessageTypeBinary": "Binary",
```
Comment (alexandrehtrb) [inline src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json:233]: Forgot translation

File: src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+  "HttpResponse/SectionTitleFailedFormat": "Ответ: Неудачный ({0})",
+  "HttpResponse/SectionTitleSuccessfulFormat": "Ответ: {0} ({1})",
+  "RequestAuth/AuthModeBasic": "Basic",
+  "RequestAuth/AuthModeBasicLoginWatermark": "(basic auth логин)",
+  "RequestAuth/AuthModeBasicPasswordWatermark": "(basic auth пароль)",
+  "RequestAuth/AuthModeBearer": "Bearer",
+  "RequestAuth/AuthModeBearerTokenWatermark": "(bearer auth токен)",
+  "RequestAuth/AuthModeNone": "None",
+  "RequestAuth/AuthModeClientCertificate":"Клиентский сертификат",
+  "RequestAuth/ClientCertificateTypeNone":"None",
```
Comment (alexandrehtrb) [inline src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json:134]: Forgot translation

File: src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+  "HttpResponse/HeaderValue": "ЗНАЧЕНИЕ",
+  "HttpResponse/SectionTitle": "Ответ",
+  "HttpResponse/SectionTitleFailedFormat": "Ответ: Неудачный ({0})",
+  "HttpResponse/SectionTitleSuccessfulFormat": "Ответ: {0} ({1})",
+  "RequestAuth/AuthModeBasic": "Basic",
+  "RequestAuth/AuthModeBasicLoginWatermark": "(basic auth логин)",
+  "RequestAuth/AuthModeBasicPasswordWatermark": "(basic auth пароль)",
+  "RequestAuth/AuthModeBearer": "Bearer",
+  "RequestAuth/AuthModeBearerTokenWatermark": "(bearer auth токен)",
+  "RequestAuth/AuthModeNone": "None",
```
Comment (alexandrehtrb) [inline src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json:132]: Forgot translation

File: src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+  "HttpRequest/BodyFormDataParamContentType": "CONTENT-TYPE",
+  "HttpRequest/BodyModeFile": "Файл",
+  "HttpRequest/BodyModeFormData": "Form Data",
+  "HttpRequest/BodyModeNone": "None",
+  "HttpRequest/BodyModeRaw": "Raw",
+  "HttpRequest/BodyModeUrlEncoded": "URL encoded",
+  "HttpRequest/BodyModeGraphQl": "Graph QL",
+  "HttpRequest/BodyGraphQlQueryWatermark": "(graph ql запрос)",
+  "HttpRequest/BodyGraphQlVariablesWatermark": "(graph ql переменные)",
+  "HttpRequest/BodyRawContentTypeWatermark": "(raw content-type here)",
```
Comment (alexandrehtrb) [inline src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json:92]: Forgot translation

File: src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+  "HttpRequest/BodyRawContentWatermark": "(raw body here)",
```
Comment (alexandrehtrb) [inline src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json:93]: Forgot translation

File: src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+  "HttpRequest/BodyFormDataParamEnabled": "ВКЛЮЧЕН",
+  "HttpRequest/BodyFormDataParamType": "ТИП",
+  "HttpRequest/BodyFormDataParamTypeText": "Текст",
+  "HttpRequest/BodyFormDataParamTypeFile": "Файл",
+  "HttpRequest/BodyFormDataParamName": "ИМЯ",
+  "HttpRequest/BodyFormDataParamTextOrSrcPath": "TEXT / PATH",
+  "HttpRequest/BodyFormDataParamContentType": "CONTENT-TYPE",
+  "HttpRequest/BodyModeFile": "Файл",
+  "HttpRequest/BodyModeFormData": "Form Data",
+  "HttpRequest/BodyModeNone": "None",
```
Comment (alexandrehtrb) [inline src/Pororoca.Desktop/Localization/ru-ru.i18n_lang.json:86]: Forgot translation

File: src/Pororoca.Desktop/Pororoca.Desktop.csproj [hunk 8]
```diff
@@ -42,6 +41,9 @@
     </AvaloniaResource>
     <AvaloniaResource Include="Assets\**" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (alexandrehtrb) [inline src/Pororoca.Desktop/Pororoca.Desktop.csproj:44]: Can you please remove these added lines?

**Review by alexandrehtrb — CHANGES_REQUESTED** [review]

**Review by alexandrehtrb — APPROVED** [review]

**Comment by alexandrehtrb** [comment]
@RVShershnev , would you like to add a russian readme too? It would be good for russian visitors.

**Comment by RVShershnev** [comment]
@alexandrehtrb Yes, I could do it.

**Pull request merged**